kod/util/system.kod

```diff
@@ -2720,6 +2733,35 @@ messages:
       return;
    }
 
+   RecreateShrineAllegiances()
+   "Resets shrine allegiances and power levels"
+   {
+      local shrine, n;
+
+      if plShrine_Allegiances <> $
+      {
+         if Length(plShrine_Allegiances) = Length(plShrines)
+         {
+            n = 1;
+
+            for shrine in plShrines
+            {
+               Send(shrine,@SetAllegiance,#allegiance=(Nth(plShrine_Allegiances,n)));
+               n = n + 1;
+            }
+         }
+         else {
```

Comment:
This is super minor but you should be consistent with brace placement.

Otherwise looks good.

Reply:
I care about the minor stuff! Thanks for spotting 👍